docs/flashbots-protect/overview.mdx

```diff
@@ -4,22 +4,21 @@ title: Overview
 
 import ProtectButtonSelector from "@site/src/components/ProtectButtonSelector"
 
-Flashbots Protect makes it easy for everyday users and developers to use Flashbots for frontrunning protection. We abstract away the complexity of submitting bundles to the Flashbots Auction and make integrating as simple as adding a URL to MetaMask.
+Flashbots Protect makes it easy for anyone to protect themselves against frontrunning and sandwich attacks by routing transactions to the Flashbots Matchmaker, instead of the public mempool. The experience for the end user is the same as before, just with added benefits.
```

Comment:
I would suggest finishing this sentence after "sandwich attacks" and not including the mention of the Matchmaker/Node.